plugin-viewport/src/test/kotlin/com/mapbox/maps/plugin/viewport/transition/MapboxViewportTransitionFactoryTest.kt

```diff
@@ -0,0 +1,161 @@
+package com.mapbox.maps.plugin.viewport.transition
+
+import android.animation.AnimatorSet
+import com.mapbox.maps.CameraOptions
+import com.mapbox.maps.plugin.animation.CameraAnimationsPlugin
+import com.mapbox.maps.plugin.animation.CameraAnimatorOptions
+import com.mapbox.maps.plugin.animation.camera
+import com.mapbox.maps.plugin.delegates.MapCameraManagerDelegate
+import com.mapbox.maps.plugin.delegates.MapDelegateProvider
+import com.mapbox.maps.plugin.delegates.MapPluginProviderDelegate
+import com.mapbox.maps.plugin.viewport.DEFAULT_FRAME_ANIMATION_DURATION_MS
+import com.mapbox.maps.plugin.viewport.DEFAULT_STATE_TRANSITION_MAX_DURATION_MS
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.slot
+import io.mockk.unmockkStatic
+import io.mockk.verify
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class MapboxViewportTransitionFactoryTest {
+  private val mapCameraDelegate = mockk<MapCameraManagerDelegate>()
+  private val mapPluginProviderDelegate = mockk<MapPluginProviderDelegate>(relaxed = true)
+  private val cameraPlugin = mockk<CameraAnimationsPlugin>()
+  private val delegateProvider = mockk<MapDelegateProvider>(relaxed = true)
+  private val animatorSet = mockk<AnimatorSet>()
+  private val constrainedSet = mockk<AnimatorSet>()
+  private lateinit var transitionsFactory: MapboxViewportTransitionFactory
+
+  @Before
+  fun setup() {
+    mockkStatic("com.mapbox.maps.plugin.viewport.transition.TransitionUtilsKt")
```

Comment:
maybe move to constant?
```suggestion
    mockkStatic(TRANSITION_UTILS)
```
same for `CameraAnimationsUtils`